broker/models.py

```diff
@@ -173,6 +173,8 @@ def __repr__(self):
 
 class CDNDedicatedWAFServiceInstance(CDNServiceInstance):
     dedicated_waf_web_acl_arn = mapped_column(db.String)
+    dedicated_waf_web_acl_id = mapped_column(db.String)
+    dedicated_waf_web_acl_name = mapped_column(db.String)
```

Comment:
there should be a migration for this, right?

Reply:
Yeah, I forgot that those exist